src/lightning/fabric/utilities/rank_zero.py

```diff
@@ -50,6 +52,34 @@ def _get_rank(
     return None
 
 
+if not _UTILITIES_GREATER_EQUAL_0_10:
```

Comment:
why don't we just bump the min version?

Reply:
This piece of code is unreleased on utilities. I don't want to block this until there's an utilities release. I needed this to fix the mypy job